mmengine/runner/log_processor.py

```diff
@@ -140,20 +140,40 @@ def get_log_after_iter(self, runner, batch_idx: int,
         #  train: Epoch [5/10000] ... (divided by `max_iter`)
         #  val/test: Epoch [5/2000] ... (divided by length of dataloader)
         if self.by_epoch:
+            # Align the iteration log:
+            # Epoch(train)  [9][010/270]
+            # ...               ||| |||
+            # Epoch(train)  [9][100/270]
+            max_dataloader_len = len(current_loop.dataloader)
+            max_str_len = len(str(max_dataloader_len))
+            cur_iter_str = str(cur_iter).rjust(max_str_len)
+
             if mode in ['train', 'val']:
+                # Right Align the epoch log:
+                # Epoch(train)   [9][100/270]
+                # ...             ||
+                # Epoch(train) [100][100/270]
                 cur_epoch = self._get_epoch(runner, mode)
+                max_epochs = runner.max_epochs
+                max_str_len = len(str(max_epochs)) + 3
```

Comment:
The `max_str_len` in line 148 means the length of dataloader but `max_epochs + 3` here, which will confuse users.

Reply:
3 means the three characters: "[", "]",  and " " occupied in " [{max_epochs}]"